Car.h

```diff
@@ -1,13 +1,16 @@
 #pragma once
 #include "Vehicle.h"
 
-struct Car : public Vehicle
+struct Car : Vehicle
 {
     Car(const std::string& s);
     
+    ~Car() override = default;
+    Car(const Car&) = default;
+    Car& operator=(const Car&) = default;
```

Comment:
It's interesting to note that while these functions were added, these don't actually need to be added to the class because of the `tryToEvade` member function that overrides the base class implementation below.  Adding this `tryToEvade` function will clear the vtable warning because it is defined outside of the class.